### ISSUE TITLE
feat: migrate website to `hugo` (or other static site generator) to avoid loading times associated with us querying the GitHub API and ratelimits

### ISSUE THREAD

**Issue by yurijmikhalevich** [move-fast-and-break-things/mfbt.community#57]
Currently, we hit GitHub API every time we load the page to render the list of community members.

This has two issues:
- it takes some time to load and render them
- we get ratelimited by GitHub pretty quickly

Pre-rendering the website would be awesome for solving this.

To do this, we need to:
- migrate the website to the static site generator
  - I prefer [hugo](https://gohugo.io/), but open to suggestions
- ensure that we fetch GitHub API only during the static site generation (instead of during the page loading)
- add a GitHub Action that will automatically build & deploy the static website on every push to the `main` branch

**Comment by LoayTarek5** [comment]
Hi @yurijmikhalevich i hope you're doing well. I'm really excited to work on this issue. could you assign it to me?
I am one that worked on that issue #6

**Comment by yurijmikhalevich** [comment]
@LoayTarek5, hi! Thank you for reaching out. Do you have experience working with static site generators? Which ones did you work with?

**Comment by LoayTarek5** [comment]
To be honest, this is the first time. But I searched a little bit about this problem, and it seems easy to solve. Could you let me try to solve it?

**Comment by yurijmikhalevich** [comment]
@LoayTarek5, of course. If you are open to trying and learning, it can be a great issue to tackle! Thank you for taking it 🙌

**Comment by LoayTarek5** [comment]
@yurijmikhalevich , Thanks for trusting me. I will do my best and ask you any questions after researching.

**Comment by yurijmikhalevich** [comment]
@LoayTarek5, sounds good! Thank **you**!

**Comment by LoayTarek5** [comment]
Hi @yurijmikhalevich, Unfortunately, this month i am so busy 
i have assignments, quizzes and exams for the university
so I am not able to work on this issue this month
if you want to assign it to another one, it is up to you

**Comment by yurijmikhalevich** [comment]
@LoayTarek5, thank you for letting me know! I have unassigned it from you for now. Feel free to reach out later when you have time 😄

**Comment by LoayTarek5** [comment]
@yurijmikhalevich, for sure❤
